src/App.js

```diff
@@ -30,13 +32,13 @@ function App() {
           )}
           <div className="my-events-link">
             {state.contact.isLoggedIn ? (
-              <Link to="/my-events" className="event-button">
+              <Link to={myEventsPath} className="event-button">
                 View my Events
               </Link>
             ) : (
               <a
                 style={{ textDecoration: 'none' }}
-                href="/_hcms/mem/login?redirect_url=/my-events"
+                href={`/_hcms/mem/login?redirect_url=/${my_events_page}`}
```

Comment:
should this be an absolute url?

Reply:
Hmm, I guess it probably should be relative since the rest of the app relies on relative paths.